tests/api_tests/nodes/test_views.py

```diff
@@ -1520,3 +1524,116 @@ def test_delete_deleted_public_node(self):
+        user_auth = Auth(self.user)
+        res = self.app.get(self.public_url, auth=self.user.auth)
+        assert_equal(res.status_code, 200)
+        self.public_project.add_tag("Jeff Spies", auth=user_auth)
+        assert_equal("tag_added", self.public_project.logs[-1].action)
+        self.public_project.remove_tag("Jeff Spies", auth=self.user_auth)
+        assert_equal("tag_removed", self.public_project.logs[-1].action)
+        res = self.app.get(self.public_url, auth=self.user)
+        assert_equal(res.json['data'][-1]['action'], 'tag_removed')
+        assert_equal("Jeff Spies", self.public_project.logs[-1].params['tag'])
+
+    def test_project_created(self):
+        res = self.app.get(self.public_url)
+        assert_equal(res.status_code, 200)
+        assert_equal(self.public_project.logs[0].action, "project_created")
+
+    def assert_datetime_equal(dt1, dt2, allowance=500):
+        assert_less(dt1 - dt2, dt.timedelta(milliseconds=allowance))
+
+    def test_log_create_on_public_project(self):
+        datetime.datetime.strptime('2015-07-28 21:06:34.965114', '%Y-%m-%d %H:%M:%S.%f')
+        res = self.app.get(self.public_url)
+        assert_equal(res.status_code, 200)
+        assert_datetime_equal(datetime.datetime.strptime(res.json['data'][0]['date'], "%Y-%m-%dT%H:%M:%S.%f"),
+                              self.public_project.logs[0].date)
+        assert_equal(res.json['data'][0]['action'], self.public_project.logs[0].action)
+        assert_equal(res.json['data'][0]['version'], self.public_project._version)
+        assert_equal(res.json['data'][0]['name'], self.public_project.logs[0]._name)
+
+    def test_log_create_on_private_project(self):
+        res = self.app.get(self.private_url, auth=self.user.auth)
+        assert_equal(res.status_code, 200)
+        assert_datetime_equal(datetime.datetime.strptime(res.json['data'][0]['date'], "%Y-%m-%dT%H:%M:%S.%f"),
+                              self.private_project.logs[0].date)
+        assert_equal(res.json['data'][0]['action'], self.private_project.logs[0].action)
+        assert_equal(res.json['data'][0]['version'], self.private_project.logs[0]._version)
+        assert_equal(res.json['data'][0]['name'], self.private_project.logs[0]._name)
+
+    def test_project_remove_contributor(self):
+        res = self.app.get(self.public_url, auth=self.user.auth)
+        assert_equal(res.status_code, 200)
+        self.public_project.add_contributor(self.contrib, auth=self.user_auth)
+        assert_equal('contributor_added', self.public_project.logs[-1].action)
+        self.public_project.remove_contributor(self.contrib, auth=self.user_auth)
+        assert_equal('contributor_removed', self.public_project.logs[-1].action)
+        res = self.app.get(self.public_url, auth=self.user.auth)
+        assert_equal(res.json['data'][-1]['action'], 'contributor_removed')
+
+    def test_add_addon(self):
+        res = self.app.get(self.public_url, auth=self.user.auth)
```

Comment:
You don't need this line nor the next one. We have tests for getting nodes, so no need to repeat that in these tests unless you're asserting something specific for the test.